plugins/autocertkit

```diff
@@ -550,6 +503,9 @@ def deploy_iperf(session, args):
                 i = i + 1  # Increment counter
                 if i == limit:
                     raise PluginError("Plugin Error: Cannot deploy Iperf")
+    else:
+        call = fr'''rpm -q iperf || rpm -i {INSTALL_DIR}/{IPERF_RPM}'''
```

Comment:
use `rpm -U` to support upgrade?

Reply:
No iperf pacakge in dom0 by default.

Reply:
no iperf package by default, but `-U` does nothing bad.